docs/environments/kind.md

```diff
@@ -43,6 +43,19 @@ script if you'd like to change the default:
 NUM_CLUSTERS=<num> ./scripts/create-clusters.sh
 ```
 
+The `KIND_TAG` is `v1.19.1@sha256:98cf5288864662e37115e362b23e4369c8c4a408f99cbc06e58ac30ddc721600` by default.
+Image `kindest/node:v1.19.1@sha256:98cf5288864662e37115e362b23e4369c8c4a408f99cbc06e58ac30ddc721600` is used as
+node docker image for booting the cluster.
+
+You can use `KIND_IMAGE` or `KIND_TAG` to specify the image as you want.
+```bash
+KIND_TAG=v1.18 ./scripts/create-clusters.sh
```

Comment:
I know this is an example but `v1.18` isn't a valid tag (people like to copy-paste). Let's use something like:
```suggestion
KIND_TAG=v1.18.8 ./scripts/create-clusters.sh
```